core/src/test/java/org/elasticsearch/index/query/AbstractQueryTestCase.java

```diff
@@ -158,11 +157,6 @@
     private static int queryNameId = 0;
     private static SearchModule searchModule;
 
-
-    protected static QueryShardContext queryShardContext() {
-        return queryShardContext;
```

Comment:
Its good to have it as a template for the createShardContext() method, we also need the instance member because we use it in lambdas for the mock mapping service etc...